components/README.md

```diff
@@ -15,13 +15,13 @@ This table lists the available types:
+| **controller** | A [Kubernetes Controller](https://kubernetes.io/docs/concepts/workloads/controllers/) which reacts to a standard Kubernetes resource or manages [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) resources. The component's name reflects the name of the primary resource it controls.| Function-controller |
 | **controller-manager** |A daemon that embeds all [Kubernetes Controllers](https://kubernetes.io/docs/concepts/workloads/controllers/) of a domain. Such an approach brings operational benefits in comparison to shipping all controllers separately. A `controller-manager` takes the name of the domain it belongs to. | - |
-| **operator** |is a [Kubernetes Operator](https://coreos.com/operators/) which covers the application-specific logic behind the operation of the application, such as steps to upscale a stateful application. It reacts on changes made to custom resources derived from a given [Custom Resource Definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/). It uses the name of the application it operates. | telemetry-operator |
+| **operator** |is a [Kubernetes Operator](https://coreos.com/operators/) which covers the application-specific logic behind the operation of the application, such as steps to upscale a stateful application. It reacts on changes made to custom resources derived from a given [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/). It uses the name of the application it operates. | telemetry-operator |
 | **job** | A [Kubernetes Job](https://kubernetes.io/docs/tasks/job/) which performs a task once or periodically. It uses the name of the task it performs. |istio-patch-job (not renamed yet)|
 | **proxy** | Acts as a proxy for an existing component, usually introducing a security model for this component. It uses the component's name. | - |
 | **service** | Serves an HTTP/S-based API, usually securely exposed to the public. It uses the domain name and the API it serves.| - |
-| **broker** | Implements the [Open Service Broker](https://www.openservicebrokerapi.org/) specification to enrich the Kyma Service Catalog with the services of a provider. It uses the name of the provider it integrates with.| azure-broker |
+| **broker** | Implements the [OpenServiceBroker](https://www.openservicebrokerapi.org/) specification to enrich the Kyma Service Catalog with the services of a provider. It uses the name of the provider it integrates with.| azure-broker |
```

Comment:
Are you sure OpenServiceBroker should be in camelcase?

Reply:
I'm absolutely not :D
It was one of the changes in https://github.com/kyma-project/kyma/pull/13634/files which I'm basically reverting, but I might have misunderstood why it was changed. 

In their own docs, they use blank spaces. But isn't it a resource, which we'd put in camel case? This seems like two style guides clashing.